src/io/flutter/preview/PreviewViewModelPanel.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.preview;
+
+import com.intellij.openapi.ui.popup.Balloon;
+import io.flutter.editor.PreviewViewController;
+import io.flutter.editor.PreviewViewControllerBase;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.awt.event.MouseMotionListener;
+
+/**
+ * Class that provides the glue to render the preview view in a regular JPanel.
+ */
+public class PreviewViewModelPanel extends JPanel {
```

Comment:
same here: rename class, fields, comments to use DeviceMirror instead of PreviewView

Reply:
added this to https://github.com/flutter/flutter-intellij/issues/4162